bundles/org.openhab.binding.venstarthermostat/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -21,6 +21,63 @@
 			<channel id="systemStateRaw" typeId="systemStateRaw"/>
 			<channel id="awayMode" typeId="awayMode"/>
 			<channel id="awayModeRaw" typeId="awayModeRaw"/>
+			<channel id="fanMode" typeId="fanMode"/>
+			<channel id="fanModeRaw" typeId="fanModeRaw"/>
+			<channel id="fanState" typeId="fanState"/>
+			<channel id="fanStateRaw" typeId="fanStateRaw"/>
+			<channel id="scheduleMode" typeId="scheduleMode"/>
+			<channel id="scheduleModeRaw" typeId="scheduleModeRaw"/>
+			<channel id="schedulePart" typeId="schedulePart"/>
+			<channel id="timestampDay0" typeId="timestampRuntime"/>
+			<channel id="timestampDay1" typeId="timestampRuntime"/>
+			<channel id="timestampDay2" typeId="timestampRuntime"/>
+			<channel id="timestampDay3" typeId="timestampRuntime"/>
+			<channel id="timestampDay4" typeId="timestampRuntime"/>
+			<channel id="timestampDay5" typeId="timestampRuntime"/>
+			<channel id="timestampDay6" typeId="timestampRuntime"/>
+			<channel id="heat1RuntimeDay0" typeId="heat1Runtime"/>
+			<channel id="heat1RuntimeDay1" typeId="heat1Runtime"/>
+			<channel id="heat1RuntimeDay2" typeId="heat1Runtime"/>
+			<channel id="heat1RuntimeDay3" typeId="heat1Runtime"/>
+			<channel id="heat1RuntimeDay4" typeId="heat1Runtime"/>
+			<channel id="heat1RuntimeDay5" typeId="heat1Runtime"/>
+			<channel id="heat1RuntimeDay6" typeId="heat1Runtime"/>
+			<channel id="heat2RuntimeDay0" typeId="heat2Runtime"/>
+			<channel id="heat2RuntimeDay1" typeId="heat2Runtime"/>
+			<channel id="heat2RuntimeDay2" typeId="heat2Runtime"/>
+			<channel id="heat2RuntimeDay3" typeId="heat2Runtime"/>
+			<channel id="heat2RuntimeDay4" typeId="heat2Runtime"/>
+			<channel id="heat2RuntimeDay5" typeId="heat2Runtime"/>
+			<channel id="heat2RuntimeDay6" typeId="heat2Runtime"/>
+			<channel id="cool1RuntimeDay0" typeId="cool1Runtime"/>
+			<channel id="cool1RuntimeDay1" typeId="cool1Runtime"/>
+			<channel id="cool1RuntimeDay2" typeId="cool1Runtime"/>
+			<channel id="cool1RuntimeDay3" typeId="cool1Runtime"/>
+			<channel id="cool1RuntimeDay4" typeId="cool1Runtime"/>
+			<channel id="cool1RuntimeDay5" typeId="cool1Runtime"/>
+			<channel id="cool1RuntimeDay6" typeId="cool1Runtime"/>
+			<channel id="cool2RuntimeDay0" typeId="cool2Runtime"/>
```

Comment:
You forgot channels cool2RuntimeDay1 to cool2RuntimeDay6

Reply:
OK. Silly me.